src/snowflake/cli/api/project/schemas/updatable_model.py

```diff
@@ -187,7 +187,8 @@ def DiscriminatorField(*args, **kwargs):  # noqa N802
     When this `DiscriminatorField` is used on a pydantic attribute,
     we will not allow templating on it.
     """
-    return Field(is_discriminator_field=True, *args, **kwargs)
+    extra = dict(is_discriminator_field=True)
```

Comment:
Could you update changes description? I figured what happened in this PR by commit title :P

Reply:
updated

Reply:
Thanks!